src/foam/u2/mlang/Pie.js

```diff
@@ -0,0 +1,38 @@
+foam.CLASS({
+  package: 'foam.u2.mlang',
+  name: 'Pie',
+  extends: 'foam.mlang.sink.GroupBy',
+  requires: [
+    'foam.graphics.DataSource',
+    'foam.graphics.PieGraph',
+  ],
+  properties: [
+    {
+      name: 'graphArgs',
+      value: {
```

Comment:
This will make it harder to configure. Make individual properties. X and Y can be expressions of radius if not set.

Reply:
Why would this make it harder to configure? If you want to customize these args you just pass in a map to clobber this one and that map will make its way to the view.

Reply:
The PieGraph could definitely use some work. It doesn't have very reasonable defaults and getting the radius, x/y, margin, and height/width right is very manual right now and I like to think we can make this easier to use. Once that's all fixed up, I should be able to ditch the default value here completely and a developer can trust it'll render fine without any tweak, but have the option to tweak if they desire.

Reply:
If I only want to change the radius but leave the colours or margin the same, then how would I do this? Also, this doesn't work as well from a FOAM tag.

Reply:
Oh gotcha. Yeah you're right about that but I do plan on removing these defaults from here completely once the PieGraph has reasonable defaults. That should solve the issue that you have with this right?

Reply:
Even with better defaults it would make sense to split these in to separate properties so its easier to only override the ones you want.  As we just discussed in person :)

Reply:
Done. I think longer term we might want to have a separate model that the Pie mlang and PieGraph pull in as traits but for now separate args will do the trick.